lib/adiwg/mdtranslator/readers/iso19115_3/modules/module_iso19115_3.rb

```diff
@@ -84,7 +84,7 @@ def self.find_contact_by_name(contactName)
 
                # add new contact to contacts array
                def self.add_contact(name, isOrg)
-                  contactId = find_contact_by_name(name)
+                  contactId = find_contact_by_name(name) unless @contacts.nil?
```

Comment:
@rshewitt note this change I had to put in to make tests pass. Oddly enough this isn't necessary with the DCAT writer changes included. That part doesn't make sense to me.

Reply:
https://github.com/GSA/mdTranslator/pull/57/commits/64c7bfd988193c1efc09bc7723a9a03f6686087b

Reply:
this didn't fix it. but the new commit did. https://github.com/GSA/mdTranslator/pull/57/commits/ee71902dbb8670dc4c8fadc14671813affcdd584

Reply:
can you put a `todo: revisit` or something on these edits? contacts are an important structure and need to be handled carefully

Reply:
i'll do it. it's fine. i want this merged.